### DISCUSSION TITLE
Home Assistant configuration

### DISCUSSION THREAD

**Discussion by benoit-cty** [mathieucarbou/YaSolR#19] [General & Support]
Hello,

In the autodiscovery channel `homeassistant/discovery/sensor/yasolr_2D1B18/grid_energy/config` , the `grid_energy` is defined as a "measurement", isn't it a [total_increasing](https://developers.home-assistant.io/blog/2021/08/16/state_class_total/) ?

**Comment by mathieucarbou** [comment]
Thanks! That was a copy paster error for 3 energy metrics. I am fixing it right now!

**Comment by mathieucarbou** [comment]
Moved to issue https://github.com/mathieucarbou/YaSolR-OSS/issues/20

**Comment by mathieucarbou** [comment]
Hello @benoit-cty ;-) 
To thank your for your testing and sponsoring I have you access to download the pro version (https://github.com/mathieucarbou/YaSolR-Pro). The only difference is in the UI  ;-)
